### PR TITLE
Adds copy_test_data to more tests (tests/test_normalize_by_median.py and tests/test_scripts.py)

### PR DESCRIPTION
**Pull request by NBKingsley**
- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make clean diff-cover` If it introduces new functionality in
  `scripts/` is it tested?
- [x] `make format diff_pylint_report cppcheck doc pydocstyle` Is it well
  formatted?
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the `ChangeLog`?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Do the changes respect streaming IO? (Are they
  tested for streaming IO?)
- [x] Is the Copyright year up to date?

…cripts.py to use the copy_test_data function.

### CODE REVIEW

**Comment by standage** [comment]
Excellent. As soon as the continuous integration tests finish, I'll merge the pull request.

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+2016-11-15 Nicole Kingsley <nbkingsley@ucdavis.edu>
+
+   * test_normalize_by_median.py, test_scripts.py:
```
Comment (ctb) [inline ChangeLog]: please put full path in -- e.g. `tests/{test_normalize_by_median.py,test_scripts.py}`

**Comment by ctb** [comment]
suggest updating the pull request name to be more succinct and precise ("use copy_test_data in some more tests")

**Comment by betatim** [comment]
[Link to travis output](https://travis-ci.org/dib-lab/khmer/jobs/176222435#L1791) that shows what is failing.

Looks like the problem is that the scripts check you aren't trying to run them on the same file twice. Before we copied the same file to two different names to trick that check. Maybe `copy_test_data` needs to grow an optional argument that allows you to make the new filename unique.

**Comment by standage** [comment]
@NBKingsley It looks like the ChangeLog has some leftovers from handling the merge conflict. :-) @shannonekj and I discussed this the other day, and I'd be happy to chat about it in person when I'm in the office.

**Comment by standage** [comment]
@NBKingsley It turns out that I can't push to your fork. It looks like you're going to have to merge and push again. Sorry!

**Comment by standage** [comment]
Never mind, GitHub has a new shiny button that allowed me to update it from the web!

**Comment by standage** [comment]
Awesome! Thanks for your contribution @NBKingsley!

**Comment by betatim** [comment]
Congrats on your first PR!

**Comment by NBKingsley** [comment]
Thanks @standage for all your assistance! And thanks @betatim for the suggestion about the test failure earlier on! So nice to have this issue completed!

**Pull request merged**